docs/.vuepress/config.js

```diff
@@ -112,6 +112,10 @@ module.exports = {
                 text: 'TiDev Twitter',
                 link: 'https://twitter.com/tidevio'
               },
+              {
+                text: 'GitHub Sponsors',
+                link: 'https://github.com/sponsors/tidev/'
+              },
               {
                 text: 'Donate',
```

Comment:
Perhaps name this `Librapay donation` or so. Then it's clear right away.

Reply:
<img width="266" alt="Bildschirmfoto 2022-05-21 um 19 18 21" src="https://user-images.githubusercontent.com/1126641/169662449-7980265f-7889-4a14-97f9-2f207371fd46.png">

Reply:
looks good :+1: